modules/ITK/cli/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+if(MSVC11) #i.e. Visual Studio 2012
+	# Fix for VS2012 that has _VARIADIC_MAX set to 10. Don't set too high because it increases compiler memory usage / compile-time.
+	add_definitions( -D_VARIADIC_MAX=10 )
```

Comment:
Yes, this code can already be found in the CMake file for the ITK Examples
https://github.com/statismo/statismo/blob/master/modules/ITK/examples/CMakeLists.txt.
We could move it up one level to the CMakeLists.txt of the ITK module, to avoid the at least for the ITK Module. 

It is also defined in the config input file:
https://github.com/statismo/statismo/blob/master/statismoConfig.cmake.in
I don't know if there is an easy (or idiomatic) way to factor all these platform specific settings into a common file?